src/js/elements/mapchip/add_filter_button.js

```diff
@@ -0,0 +1,28 @@
+import { Observable } from "../../utils";
+
+export class AddFilterButton extends Observable {
+    static EVENTS = {
+        "clicked": "addButton.clicked"
+    }
+    constructor() {
```

Comment:
Can you add a very simple model class here that keeps the state of whether it's enabled or disabled? Remember to make sure that the model fires off an event when the state is changed so that the controller can catch it and call `this.button.addClass('disabled')` or` this.button.removeClass('disabled'); `